api/views.py

```diff
@@ -32,66 +73,94 @@ def default(self, obj):
           return list(obj)
        return json.JSONEncoder.default(self, obj)
 
+def get_post_query(request):
```

Comment:
This could be put in the Post Model or view i guess. Better then having to modify this query in multiple locations